src/components/AccessibleTooltip.tsx

```diff
@@ -0,0 +1,66 @@
+import { Tooltip } from "antd";
+import { TooltipPlacement } from "antd/es/tooltip";
+import React, { PropsWithChildren, ReactElement, cloneElement, forwardRef, useEffect, useRef, useState } from "react";
+
+type AccessibleTooltipProps = {
+  disabled?: boolean;
+  title?: string;
+  placement?: TooltipPlacement;
+};
+
+/**
+ * Layout-safe, focus-responsive tooltip! This is a wrapper around the Ant Tooltip,
+ * fixing some of the biggest issues with it.
```

Comment:
could be interesting to check antd's issues and see if others have brought it up

Reply:
🙃 just realized there's a focus trigger mode in Tooltip that would have worked for this use case actually and I did not need to rewrite this from scratch

The layout issue still persists though when disabling tooltips, but this might let me fix the bug I noted with Dropdown...

Reply:
This was actually a great comment because it turns out the issue I was having with Tooltips changing layout doesn't happen anymore? I'm not sure if it got fixed in an update ((or if Ant is just gaslighting me)) but I've removed `AccessibleTooltip.tsx` and simplified code in a few places.